internal/environment/command_create.go

```diff
@@ -17,15 +16,28 @@ func (c *command) newCreateCommand() *cobra.Command {
 		RunE:  c.create,
 	}
 
+	cmd.Flags().String("stream-governance-package", "ESSENTIALS",
+		"Stream Governance package. \"ESSENTIALS\" (default) or \"ADVANCED\"")
```

Comment:
* The CLI library we use should specify which one is the default for you
* Please support lowercase input since that's the format that all of our examples and documentation uses
* You can use `utils.ArrayToCommaDelimitedString` here
```suggestion
		`Stream Governance package. "essentials" or "advanced"`)
```